src/styles/custom.css

```diff
@@ -0,0 +1,38 @@
+/* ========================= 
+    NavBar 
+========================= */
+.c-language-selector {
+    @apply pl-6 relative;
+}
+
+.c-language-menu {
+    @apply overflow-hidden;
+    top: 36px;
+    right: -20px;
+    max-height: 0;
+    animation: full-height-500 0.4s 1 ease-in-out both;
+}
+
+.c-language-selector:before {
+    content: "";
+    position: absolute;
+    width: 20px;
+    height: 20px;
+    top: 1px;
+    left: 0;
+    background-position: center;
+    background-size: contain;
+    background-image: url(../images/icons/language-selector.svg);
+}
+
+/* ========================= 
+    List Card 
+========================= */
+@media (min-width: 1024px) {
```

Comment:
You can try tailwind's `@responsive` and `@variant` directives here:
- https://tailwindcss.com/docs/adding-new-utilities#generating-responsive-variants
- https://tailwindcss.com/docs/adding-new-utilities/#generating-pseudo-class-variants

If you then name the class something like `c-elevate` to describe its effect, rather than describing where it's used, you can write `<div className="lg:hover:c-elevate"> ...` to apply it to the card.

The main benefit is getting away from specifying pixel values in CSS (e.g. `min-width: 1024px`), so responsive breakpoints are handled consistently, but it also makes the elevation styling reusable. 

Something like:
```
@responsive {
  @variants hover {
    .c-elevate {
      transform: translateY(-5px);
      box-shadow: 3px 7px 12px #e4e4e4;
    }
  }
}
```